examples/python/monitoring/05_error_handling_monitoring.py

```
@@ -0,0 +1,195 @@
+- Recovery mechanism performance
+- Exception handling with monitoring
+
+Shows how to monitor performance when things go wrong.
+"""
+
+import random
+import time
+from praisonaiagents import Agent
+from praisonaiagents.telemetry import (
+    monitor_function, track_api_call, get_function_stats,
+    get_api_stats, performance_monitor
```

Ruff (0.12.2)

`praisonaiagents.telemetry.performance_monitor` imported but unused

Remove unused import: `praisonaiagents.telemetry.performance_monitor`

(F401)